netci.groovy

```diff
@@ -6,7 +6,7 @@ def project = GithubProject
 def branch = GithubBranchName
 
 [true, false].each { isPR ->
-    ['Debug', 'Release', 'Desktop.Debug', 'Desktop.Release'].each { configuration ->
+    ['Lab.Debug', 'Lab.Release', 'Desktop.Debug', 'Desktop.Release'].each { configuration ->
```

Comment:
This will build the Lab version of the CoreCLR config, but the non-lab version of the Desktop config.  The Desktop configs don't have a separate "Lab" version.  You should be able to leave this line alone and specify "/p:Lab=true" in the commandline a few lines below here, which should work for both versions.